index.html

```diff
@@ -0,0 +1,201 @@
+
+The final component of the URL's `"/"`-separated `path` **must** be a valid [SemVer specifier](#semver-specifiers) identifying the version of the spec in use. Everything prior to the version specifier is the spec's *base URL*. Processors **should** treat the spec's base URL as an opaque string identifying the specification.
+
+This spec assigns no meaning to additional URL components, such as the query string or fragment. Processors **should** ignore any meaningless URL components when parsing the base URL and semver specifier.
+
+```graphql
+#::[`@using` directive with a query string]
+schema
+  @using(spec: "https://spec.example.com/exampleSpec/1?unrelated=value")
+  { ... }
+```
+
+This directive:
+- Declares that the document conforms to a spec which can be found at `https://spec.example.com/exampleSpec/1`
+- Has a base URL of `https://spec.example.com/exampleSpec`
+- Has a [semver specifier](#semver-specifier) of `1`
+- Has a query string of `"unrelated=value":
+  - This has no effect unless `exampleSpec` assigns it specific meaning.
+
+## Semantic Versioning
+
+Specifications **should** provide a semantic version number.
+
+### SemVer Specifiers
+
+The final `/`-separated path component **must** be a semver specifier. Semver specifiers take the form *major*.*minor*.*patch*-*prerelease*. The minor, patch, and pre-release specifiers are all optional. Examples:
+
+```
+#::[Valid SemVer specifiers]
+1
+1.2
+1.2.3-alpha
+2.2.3-alpha.2
+```
+
+### SemVer selection
+
+Processors **should** select the highest available spec implementation which is compatible with the requested version according to the [SemVer 2.0.0](https://semver.org/spec/v2.0.0.html) rules for version compatibility. For example, consider this directive:
+
+```graphql
+#::[`@using` version 2.x.y]
+schema @using(spec: "https://specs.example.com/exampleSpec/2")
+```
+
+Upon seeing this directive, processors which support the `https://specs.example.com/exampleSpec` spec **should** activate a version of their implementation which is compatible with major version 2.
+
+```graphql
+#::[`@using` version 3.1.1-alpha]
+schema @using(spec: "https://specs.example.com/exampleSpec/3.1.1-alpha")
+```
+
+This selects versions in the `3.1.1-alpha` prerelease series. For example, a processor may satisfy this requirement by activating version `3.1.1-alpha.2` of `exampleSpec`.
+
+### Why is versioning in the URL, not a directive argument?
+
+The version is in the URL because when a human reader visits the URL, we would like them to be taken to the documentation for the *version of the specification in use by this document*. Putting the version information in a separate argument to the `@using` directive would prevent this.
+
+### Backus-Naur Form for SemVer specifier
+
+```text
+#::[BNF grammar for SemVer specifiers]
+(valid specifier) ::= "v" (partial semver)
+(partial semver) ::= (major)
```

Comment:
Copy/paste of [this comment](https://github.com/apollographql/specs/pull/1/files#r527821262).

nit: Could we use angle brackets here instead of parentheses? Parentheses can be used for grouping in BNF grammars, so it's a bit clearer to use angle brackets for BNF classes instead.

Reply:
done.